cmd/nerdctl/volume_ls_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/tabutil"
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestVolumeLs(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+	testutil.DockerIncompatible(t)
+
+	var vol1, vol2, vol3 = tID + "vol-1", tID + "vol-2", tID + "empty"
+	base.Cmd("volume", "create", vol1).AssertOK()
+	defer base.Cmd("volume", "rm", "-f", vol1).Run()
+
+	base.Cmd("volume", "create", vol2).AssertOK()
+	defer base.Cmd("volume", "rm", "-f", vol2).Run()
+
+	base.Cmd("volume", "create", vol3).AssertOK()
+	defer base.Cmd("volume", "rm", "-f", vol3).Run()
+
+	createFileWithSize(t, vol1, 102400)
+	createFileWithSize(t, vol2, 204800)
+
+	base.Cmd("volume", "ls", "--size").AssertOutWithFunc(func(stdout string) error {
+		var lines = strings.Split(strings.TrimSpace(stdout), "\n")
```

Comment:
Please check length of lines

Reply:
This is not required since we already have numMatches and are parsing header.

Reply:
No, `lines[0]` maybe cause panic